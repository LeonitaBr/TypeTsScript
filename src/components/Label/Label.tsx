import React, {memo} from 'react';
import {Text, View} from 'react-native';
import styles from './style';
interface ButtonProps {
  title: string;
  labelStyles?: object;
}

const Label: React.FC<ButtonProps> = ({title, labelStyles}) => {
  return (
    <View>
      <Text style={{...styles.title, ...labelStyles}}>{title}</Text>
    </View>
  );
};
export default memo(Label);
