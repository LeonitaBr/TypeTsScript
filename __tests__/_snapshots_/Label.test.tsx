import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../../src/components/Label/Label';
describe('Some component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Label title={'AAAAA'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
