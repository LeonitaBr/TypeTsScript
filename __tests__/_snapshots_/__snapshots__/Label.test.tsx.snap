// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some component renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 18,
      }
    }
  >
    AAAAA
  </Text>
</View>
`;
